docker-image/v1.15/arm64/debian-opensearch/Gemfile.lock

```diff
@@ -4,19 +4,19 @@ GEM
     addressable (2.8.1)
       public_suffix (>= 2.0.2, < 6.0)
     aws-eventstream (1.2.0)
-    aws-partitions (1.621.0)
-    aws-sdk-core (3.134.0)
+    aws-partitions (1.646.0)
+    aws-sdk-core (3.160.0)
```

Comment:
You don't need update Gemfile.lock manually, it will be updated automatically on every releases.
FYI: https://github.com/fluent/fluentd-kubernetes-daemonset/blob/master/MAINTAINING.md#updating

Reply:
I ran `make src-all` with all X86_IMAGES commented except opensearch.

I wasn't 100% sure if that was an expected step of the PR. I'll rollback that change.

Reply:
Done